src/Stratis.Bitcoin.Features.Miner/MinerSettings.cs

```diff
@@ -92,13 +115,16 @@ public static void PrintHelp(Network mainNet)
             NodeSettings defaults = NodeSettings.Default();
             var builder = new StringBuilder();
 
-            builder.AppendLine("-mine=<0 or 1>            Enable POW mining.");
-            builder.AppendLine("-stake=<0 or 1>           Enable POS.");
-            builder.AppendLine("-mineaddress=<string>     The address to use for mining (empty string to select an address from the wallet).");
-            builder.AppendLine("-walletname=<string>      The wallet name to use when staking.");
-            builder.AppendLine("-walletpassword=<string>  Password to unlock the wallet.");
-            builder.AppendLine("-blockmaxsize=<number>    Maximum block size (in bytes) for the miner to generate.");
-            builder.AppendLine("-blockmaxweight=<number>  Maximum block weight (in weight units) for the miner to generate.");
+            builder.AppendLine("-mine=<0 or 1>                      Enable POW mining.");
+            builder.AppendLine("-stake=<0 or 1>                     Enable POS.");
```

Comment:
This is not enabled in command link only in api/rpc to avoid passing the password.